### ISSUE TITLE
Struggling to implement a few custom classes- couldn't find a worhtwile solution 

### ISSUE THREAD

**Issue by enesyue** [gehrisandro/tailwind-merge-laravel#16]
Hey there,

I'm sorry for the troubles, but I'm kind of struggling to implement a couple of extra classes based on a tw Plugin called "tailwind-bootstrap-grid"
It implements Bootstraps grid-system within Tailwind (ref: https://github.com/karolis-sh/tailwind-bootstrap-grid/blob/main/lib/index.js)
I've tried couple of things, but for some odd reason it doesn't seem to work at all.
Trying to make twMerge filter x:col-*, but it doesn't recognize those (obviously, since they aren't part of tailwind) and just outputs both values.
And for some reason I'm somewhat struggling to appened the functionality within the packages classGroups param.
Anyone here could hand a helping hand?

(sub)-component looks something like this:
<div {{ $attributes->twMerge(['class' => 'swiper-slide relative col-10']) }}>
    {{ $slot }}
</div>

tysm 

### **tl:dr**
I'm using a set of custom classes, such as:

col-1, col-2, col-3... col-12
sm:col-1... sm:col-12
md:col-1... md:col-12
lg:col-1...

(*not to confuse with col-span)

this is the tailwind-config markup for the x:col-* classes:
```
...columns.map((size) => ({
  [`.col-${size}`]: {
    flex: '0 0 auto',
    width: `${(100 / gridColumns) * size}%`,
  },
})),
```

I wasn't able to adjust the config and would appreciate any form of help
